Add clarification for [Wrapper for Traits] section (#188)
paritytech/ink-docs: docs/third-party-tools/openbrush.md

### Wrapper around Traits

Traditionally, if you want to do a cross-contract calling in ink!,
you need to import the other contract as a dependency in the calling contract's project. 

> OpenBrush simplifies the process with the wrapper around traits. 

Reviewer: Can you clarify here: 
* What is meant with "the wrapper"?
* Why does it need to be simplified in the first place?
* Is it just a simplification or does "The Wrapper" add guarantees?

> ### Wrapper around Traits

Reviewer: ```suggestion
### Wrapper around Traits: `#[openbrush::wrapper]`
```

Traditionally, if you want to do a cross-contract calling in ink!,

> you need to import the other contract as a dependency in the calling contract's project. 

Author: What's the actual benefit for OB's wrapper then? If we have our own dynamic wrapper?

> you need to import the other contract as a dependency in the calling contract's project. 

Reviewer 2: 1. It supports several traits(You can write `dyn Erc20 + Ownable + Erc721`).
1. The IDEs support hinting and highlighting=)

But with the new trait system that we will develop later, there will be no difference=)

> OpenBrush simplifies the process with the wrapper around traits.

Reviewer: ```suggestion
OpenBrush simplifies the process with a wrapper around traits.
```

> If you know that the other contract implements a trait, 

Reviewer 2: The trait should be defined via `#[openbrush::trait_definition]`.

you only need that trait definition and the address of the other contract 
in order to call a method from the deployed third-party contract.

> The wrapper is an attribute macro for traits that can be used to automatically generate 

Reviewer: ```suggestion
The wrapper is an attribute macro named `#[openbrush::wrapper]` for traits.  It can be
used to automatically generate 
```

> a callable `{Contract}Ref` implementation.

Reviewer 2: Actually, the name is defined by the user because the syntax is:
```rust
// You can create wrapper in the place where you defined the trait
// Or if you import **everything** from the file where you define trait
#[openbrush::wrapper]
type Trait1Ref = dyn Trait1;
```

> The benefits of such mechanism is a functional polymorphism.

Reviewer: But don't you also have that with our add-contract-as-dependency mechanism?

Instead of working with the concrete implementation of the trait as a dependency
in you contract, you only interact with its interface. 

> That means that you do not need to import other contract's as dependencies

Reviewer: ```suggestion
That means that you do not need to import other contracts as dependencies
```
